pkg/operator/ceph/controller/spec.go

```diff
@@ -653,8 +655,27 @@ func PrivilegedContext(runAsRoot bool) *v1.SecurityContext {
 	return sec
 }
 
-// LogCollectorContainer runs a cron job to rotate logs
+// LogCollectorContainer rotate logs
 func LogCollectorContainer(daemonID, ns string, c cephv1.ClusterSpec) *v1.Container {
+
+	maxLogSize := "500M"
+	if c.LogCollector.MaxLogSize != nil {
+		maxLogSize = fmt.Sprintf("%dM", c.LogCollector.MaxLogSize.Value()/1000/1000)
+		if maxLogSize == "0M" {
+			logger.Info("maxLogSize is 0M setting to minimum of 1M")
+			maxLogSize = "1M"
+		}
+	}
+
+	periodicity := "daily"
+	if c.LogCollector.Periodicity == "1h" || strings.Contains(c.LogCollector.Periodicity, "h") {
+		periodicity = "hourly"
+	} else if c.LogCollector.Periodicity == "24h" || c.LogCollector.Periodicity == "1d" || strings.Contains(c.LogCollector.Periodicity, "d") {
```

Comment:
```suggestion
	} else if c.LogCollector.Periodicity == "24h" || c.LogCollector.Periodicity == "1d" || strings.Contains(c.LogCollector.Periodicity, "d") {
```
same here

Reply:
We can just simplify this to any h or d will result in "daily". So basically, if they have 2h or 12h or 24h or 1d or 2d or anything like that, we just default to daily.
```suggestion
	} else if c.LogCollector.Periodicity == "h" || strings.Contains(c.LogCollector.Periodicity, "d") {
```